### DISCUSSION TITLE
What am I missing with Symbol Markers?

### DISCUSSION THREAD

**Discussion by Pitchblack70** [keiwando/toonsquid#391] [Q&A]
ORIGINAL QUESTION (SEE PARTIAL ANSWER BELOW)

Hi all!  

I'm doing some lip syncing, and can't get the Select by Markers option to appear.

I create a symbol, set Markers... and can select by the Timeline - but the "Select by Markers" alternative doesn't even pop up. What am I missing?  (Probably something incredibly easy, once I know what it is.)  

I've read the handbook - it didn't help. Thanks!!  :))

--Pitch

JUST FOUND A WAY IT WORKS!

So, up until now - I've been creating markers in the project timeline. With that method, it DOESN'T show up in the properties panel. (Is that a program failure to link?)

BUT - if I go into the Library, hold the clip to edit... and name the markers THERE... THEN it shows up. Yay... I've been so confused about this. And I'm sure I'm not the only user with that issue?!?  :)

PS: Any chance of creating an auto lip sync function eventually?  :)))

**Comment by Cat-Butler** [comment]
Hi @Pitchblack70 and well done and thank you for finding and sharing the solution for other forum members.

I admit I had to read it a couple of times, but the manual does have Frame Markers as a sub-section under symbols, and the lipsync example starts with "_We [create a symbol](https://toonsquid.com/handbook/symbols/create_symbol) which will hold all of the different mouth shapes that are needed for lip-syncing._" 
**See:**  https://toonsquid.com/handbook/symbols/marker_animation/
The [official video for Frame Markers and lipsync](https://www.youtube.com/watch?v=zoWPIkSJLWs) also refers to creating a symbol (at a little past the 4-minute mark).

Yes! to a more automated lip sync feature.
Alternatively, I'd like some lipsync data import, such as .dat file import from [Rhubarb](https://github.com/DanielSWolf/rhubarb-lip-sync) or [Papagayo](https://lostmarble.com/papagayo/) or [one of its variations ](https://morevnaproject.org/papagayo-ng/)- unfortunately I don't know of any iPad equivalent, and I'm not sure there enough users who would benefit from/use this, so something built into ToonSquid would probably be better.

**Comment by generalinq** [comment]
> ORIGINAL QUESTION (SEE PARTIAL ANSWER BELOW)
> 
> Hi all!
> 
> I'm doing some lip syncing, and can't get the Select by Markers option to appear.
> 
> I create a symbol, set Markers... and can select by the Timeline - but the "Select by Markers" alternative doesn't even pop up. What am I missing? (Probably something incredibly easy, once I know what it is.)
> 
> I've read the handbook - it didn't help. Thanks!! :))
> 
> --Pitch
> 
> JUST FOUND A WAY IT WORKS!
> 
> So, up until now - I've been creating markers in the project timeline. With that method, it DOESN'T show up in the properties panel. (Is that a program failure to link?)
> 
> BUT - if I go into the Library, hold the clip to edit... and name the markers THERE... THEN it shows up. Yay... I've been so confused about this. And I'm sure I'm not the only user with that issue?!? :)
> 
> PS: Any chance of creating an auto lip sync function eventually? :)))

I had the exact same problem and realization. You weren’t the only one.
